scene/resources/convex_polygon_shape_2d.cpp

```diff
@@ -65,7 +65,7 @@ bool is_convex(const Vector<Vector2> &p_points) {
 
 void ConvexPolygonShape2D::_update_shape() {
 	Vector<Vector2> final_points = points;
-	if (Geometry2D::is_polygon_clockwise(final_points)) { //needs to be counter clockwise
+	if (!Geometry2D::is_polygon_clockwise(final_points)) { // Needs to be clockwise.
```

Comment:
```suggestion
	if (!Geometry2D::is_polygon_clockwise(final_points)) { // Needs to be counter clockwise.
```

With your added `!`, shouldn't the comment say `Needs to be counter clockwise.`?

Reply:
That's correct, thank you. Updated and rebased in 7961ddf3428b7e7322b5421078633a015862b8bf.